### PR TITLE
Initial cut at saving JIMS1058

### PR DESCRIPTION
**Pull request by colbywhite**
This just saves the file in the expected spot. As noted in the TODO list in the `README`, next would be to commit the change.

Should we close the original open-austin/project-ideas#73 and start bringing the convo to this repo?

### CODE REVIEW

**Comment by luqmaan** [comment]
How do you plan to to deploy this?

I'd prefer we use something thats compatible with AWS Lambda (Node.js, Python and Java). That way we don't have to setup a server with cron. https://aws.amazon.com/lambda/

Take a look at https://github.com/open-austin/construction-permits as an example.

**Comment by colbywhite** [comment]
How in the world did I not realize that Lambda doesn't support ruby? :doh: Well, python it is.

**Comment by luqmaan** [comment]
😘 Yeah it sucks they don't support ruby.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.swp
+.DS_Store
```
Comment (luqmaan) [inline .gitignore]: Need a newline here

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# IntelliJ Default
+Metrics/LineLength:
+  Max: 120
```
Comment (luqmaan) [inline .rubocop.yml]: Newline, although I guess this file will be going away.

**Comment by colbywhite** [comment]
@luqmaan, It's been switched to python. Enjoy. 😉 

@fileunderjeff, In response to your comment about privacy on open-austin/project-ideas#73, I rebased the one data file I had in here out. I won't start saving data until Open Austin comes to a consensus on how they want to handle it.

File: .gitignore (hunk 1 above)
Comment (colbywhite) [inline .gitignore]: FYI: this has been fixed in 7a782aa. The GitHub UI didn't like it when I rebased that one data file out.

**Pull request merged**